app/src/main/java/com/onegini/mobile/exampleapp/view/helper/NotificationMenuHelper.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016-2017 Onegini B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.onegini.mobile.exampleapp.view.helper;
+
+import android.content.Context;
+import com.onegini.mobile.exampleapp.R;
+
+public class NotificationMenuHelper {
+
+  public static String getNotificationTitle(final Context context, final int notificationsCount) {
+    if (notificationsCount == 0) {
```

Comment:
oh, this logic is already invented and incorporated into android resources. Use quantity strings: https://developer.android.com/guide/topics/resources/string-resource.html#Plurals that's to it the method will be unnecessary and imo the less code the better

Reply:
Didn't know about this feature 👍 But to be honest this seems to be to complex for a simple button, so I have simplified the implementation itself.

Reply:
BUt imo it's better to have 'the login' in resources so you don't have to pay attention to it. It will be a lot simpler imo - you just nee to define `zero` `one` and `other`